### PR TITLE
Callouts: Add GetSelectionBox

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: rts/Game/UI/MouseHandler.cpp [hunk 1]
```diff
@@ -340,7 +340,43 @@ void CMouseHandler::MousePress(int x, int y, int button)
 	}
 }
 
+/**
+ * GetSelectionBoxVertices
+ *  returns the vertices of the SelectionBox in (cam->right, cam->up)-space
+ */
+bool CMouseHandler::GetSelectionBoxVertices(float3& bl, float3& br, float3& tl, float3& tr)
```
Comment (lhog) [inline rts/Game/UI/MouseHandler.cpp:347]: Try:
`bool CMouseHandler::GetSelectionBoxVertices(float3& bl, float3& br, float3& tl, float3& tr) const`

File: rts/Game/UI/MouseHandler.h [hunk 2]
```diff
@@ -81,6 +81,8 @@ class CMouseHandler
 	/// @see ConfigHandler::ConfigNotifyCallback
 	void ConfigNotify(const std::string& key, const std::string& value);
 
+	bool GetSelectionBoxVertices(float3& bl, float3& br, float3& tl, float3& tr);
```
Comment (lhog) [inline rts/Game/UI/MouseHandler.h:84]: Try:
`bool GetSelectionBoxVertices(float3& bl, float3& br, float3& tl, float3& tr) const`

File: rts/Lua/LuaUnsyncedRead.cpp [hunk 3]
```diff
@@ -602,6 +603,35 @@ int LuaUnsyncedRead::GetMiniMapDualScreen(lua_State* L)
...
+	lua_rawseti(L, -2, 2);
+
+	lua_createtable(L, 2, 0);
+
+	lua_pushnumber(L, topRight.x);
+	lua_rawseti(L, -2, 1);
+	lua_pushnumber(L, topRight.y);
+	lua_rawseti(L, -2, 2);
+
+	return 2;
```
Comment (lhog) [inline rts/Lua/LuaUnsyncedRead.cpp:632]: Convert output to just 4 lua numbers, I don't think tables here make sense

File: rts/Game/UI/MouseHandler.cpp (hunk 1 above)
Comment (badosu) [reply rts/Game/UI/MouseHandler.cpp]: Done

File: rts/Game/UI/MouseHandler.h (hunk 2 above)
Comment (badosu) [reply rts/Game/UI/MouseHandler.h]: Done

File: rts/Lua/LuaUnsyncedRead.cpp (hunk 3 above)
Comment (badosu) [reply rts/Lua/LuaUnsyncedRead.cpp]: Done

**Comment by lhog** [comment]
Thanks

**Pull request merged**